snowpack/src/commands/install.ts

```diff
@@ -398,13 +399,15 @@ ${colors.dim(
   };
   if (Object.keys(installEntrypoints).length > 0) {
     try {
+      logger.debug(`running installer with options: ${util.format(inputOptions)}`);
```

Comment:
reasoning: The entire install function exists to create this options object that we pass to rollup, so we can debug most issues by just knowing what's in this `inputOptions` object. I'd like to start here, and then add more fine-grained logs if future issues point to us needed more detail than this.